legend-engine-server/pom.xml

```diff
@@ -570,6 +570,11 @@
             <artifactId>legend-pure-runtime-java-extension-external-json</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
```

Comment:
Have we checked sdlc pipelines with mapping tests, I think we would similar dependency in extensions-execution module?

Reply:
added sdlc PR: https://github.com/finos/legend-sdlc/pull/516